Directory.Build.props

```diff
@@ -13,6 +13,15 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <NoWarn>$(NoWarn);Uno0001</NoWarn>
+    <!--
+      Suppress 'error CsWinRT1028: Class implements WinRT interfaces but isn't marked partial. ' for generated XamlMetaDataProvider.g.cs.
+      Fix is expected to arrive in prerelease1 of Microsoft.WindowsAppSdk 1.6.
```

Comment:
WinAppSDK 1.6-preview1 just dropped today, try updating it! Then you can remove this suppression 🙂